### PR TITLE
8.0 document page environmental aspect

### PR DESCRIPTION
**Pull request by max3903**
Depends on:
- [x] #20 
- [x] OCA/Knowledge#30

### CODE REVIEW

**Comment by dreispt** [comment]
I believe the module icon should now go in a `description/` directory.

**Comment by llacroix** [comment]
@max3903  could you add the dependencies in the description of PR.

Depends on:
- [x] #20 
- [x] OCA/Knowledge#30

File: document_page_environmental_aspect/data/document_page.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
```
Comment (dreispt) [inline document_page_environmental_aspect/data/document_page.xml]: noupdate="1" ?

File: document_page_environmental_aspect/views/document_page.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<openerp>
+  <data>
+
+    <record model="ir.actions.act_window" id="action_environmental_aspects">
+      <field name="name">Environmental Aspects</field>
+      <field name="res_model">document.page</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">tree,form</field>
+      <field name="view_id" ref="document_page.view_wiki_tree"/>
+      <field name="search_view_id" ref="document_page.view_wiki_filter"/>
```
Comment (dreispt) [inline document_page_environmental_aspect/views/document_page.xml]: I would rather not force specific view_ids for better flexibility

**Comment by dreispt** [comment]
Minor notes, but overall :+1:

**Comment by max3903** [comment]
Tested on runbot

:+1:

**Pull request merged**